global-intuitive.postman_collection.json

```diff
@@ -0,0 +1,223 @@
+					"header": [],
+					"cookie": [],
+					"responseTime": 0,
+					"body": "{\n    \"count\": 1,\n    \"results\": [\n        {\n            \"suggestion\": \"Experian Data Quality, George West House, 2-3 Clapham Common North Side, London, SW4 ...\",\n            \"matched\": [\n                [\n                    67,\n                    71\n                ],\n                [\n                    46,\n                    66\n                ],\n                [\n                    42,\n                    45\n                ]\n            ],\n            \"format\": \"https://api.edq.com/capture/address/v2/format?country=GBR&id=AVzAG9qw2CTc2aQaIiu4\"\n        }\n    ]\n}"
+				}
+			]
+		},
+		{
+			"name": "Format",
+			"request": {
+				"url": {
+					"raw": "https://api.edq.com/capture/address/v2/format?country=<3 letter ISO country code>&id=<format ID>",
+					"protocol": "https",
+					"host": [
+						"api",
+						"edq",
+						"com"
+					],
+					"path": [
+						"capture",
+						"address",
+						"v2",
+						"format"
+					],
+					"query": [
+						{
+							"key": "country",
+							"value": "<3 letter ISO country code>",
+							"equals": true,
+							"description": "Must be a licensed country. See documentation for full list of supported ISO codes."
+						},
+						{
+							"key": "id",
+							"value": "<format ID>",
+							"equals": true,
+							"description": "ID is returned in the payload of the Search request. The full Format URL is returned, not just the ID."
+						}
+					],
+					"variable": []
+				},
+				"method": "GET",
+				"header": [
+					{
+						"key": "Auth-Token",
+						"value": "{{Auth-Token}}",
+						"description": ""
+					}
+				],
+				"body": {},
+				"description": "Returns the full address and component breakdown for the chosen address."
+			},
+			"response": [
+				{
+					"id": "d183cf1f-cbbf-4112-ac4d-0d5d413c01c4",
+					"name": "Format Example",
+					"originalRequest": {
+						"url": {
+							"raw": "https://api.edq.com/capture/address/v2/format?country=gbr&id=AVzAG9qw2CTc2aQaIiu4",
+							"protocol": "https",
+							"host": [
+								"api",
+								"edq",
+								"com"
+							],
+							"path": [
+								"capture",
+								"address",
+								"v2",
+								"format"
+							],
+							"query": [
+								{
+									"key": "country",
+									"value": "gbr",
+									"equals": true,
+									"description": "Must be a licensed country. See documentation for full list of supported ISO codes."
+								},
+								{
+									"key": "id",
+									"value": "AVzAG9qw2CTc2aQaIiu4",
+									"equals": true,
+									"description": "ID is returned in the payload of the Search request. The full Format URL is returned, not just the ID."
+								}
+							],
+							"variable": []
+						},
+						"method": "GET",
+						"header": [
+							{
+								"key": "Auth-Token",
+								"value": "a1b234c5-a1bc-1234-a1bc-123a4567891bc",
```

Comment:
An auth-token value - just checking this isn't a real value? Might be safer to remove this altogether? Not sure if you can put variables in examples though..